code/modules/events/_event.dm

```diff
@@ -84,6 +84,7 @@
 	triggering = TRUE
 	if (alert_observers)
 		message_admins("Random Event triggering in [RANDOM_EVENT_ADMIN_INTERVENTION_TIME] seconds: [name] (<a href='?src=[REF(src)];cancel=1'>CANCEL</a>)")
+		play_sound_to_all_admins('sound/effects/admin_alert.ogg')
```

Comment:
you should add this sound to fugitive backstory selection as well

Reply:
Done